test/bdd/pkg/waci/issue_credential_attachments.json

```diff
@@ -0,0 +1,60 @@
+[
+  {
+    "id":"e00e11d4-906d-4c88-ba72-7c66c7113a78",
```

Comment:
can we add proof to credentials/presentations dynamically in tests?
and we can also verify signature in holder applications.

@rolsonquadras any thoughts?